buildSrc/subprojects/configuration/src/main/kotlin/org/gradle/gradlebuild/java/AvailableJavaInstallations.kt

```diff
@@ -47,11 +47,7 @@ class ProbedLocalJavaInstallation(private val javaHome: File) : LocalJavaInstall
 
 
 private
-const val java7HomePropertyName = "java7Home"
-
-
-private
-const val java9HomePropertyName = "java9Home"
+const val compileJavaHomePropertyName = "java9Home"
```

Comment:
Let's keep calling this `java9HomePropertyName`. That's what it is, a property pointing the home of java 9. Later on, we can add a `java10Home` if we need Java 10 in some places or everywhere and the old build would still work.